client/src/Home.jsx

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import axios from 'axios';
+import Navbar from '../components/Navbar';
+import Sidebar from '../components/Sidebar';
+import './home.css';
+
+export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      userPlaylists: {},
+      featuredPlaylists: {},
+      recentlyPlayed: {},
+      profile: {},
+      artists: {}
+    };
+  }
+
+  componentDidMount() {
+
+    axios.get('/me/playlists')
+      .then(res => {
+        const playlists = res.data;
+        this.setState({ isLoading: false, userPlaylists: playlists });
+      });
+
+    axios.get('/browse/featured-playlists')
+      .then(res => {
+        const featuredPlaylists = res.data;
+        this.setState({ featuredPlaylists });
+      });
+
+    axios.get('/me/player/recently-played')
+      .then(res => {
+        const recentlyPlayed = res.data;
+        this.setState({ recentlyPlayed });
+      });
+
+    axios.get('/me')
+      .then(res => {
+        const profile = res.data;
+        this.setState({ profile });
+      });
+
+    axios.get('/me/following?type=artist')
+      .then(res => {
+        const artists = res.data;
+        this.setState({ artists });
+      });
```

Comment:
If you want to block on all of these, you can use Promise.all()

```js
Promise.all([
    axios.get('/me/playlists')
      .then(res => res.data),
    axios.get('/browse/featured-playlists')
      .then(res => res.data),
    axios.get('/me/player/recently-played')
      .then(res => res.data),
    axios.get('/me')
      .then(res => res.data),
    axios.get('/me/following?type=artist')
      .then(res => res.data)
]).then(([playlists, featuredPlaylists, recentlyPlayed, profile, artists]) => {
  this.setState({ playlists, featuredPlaylists, recentlyPlayed, profile, artists, isLoading: false });
})
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all